packages/gatsby-plugin-typescript/src/gatsby-node.js

```diff
@@ -33,6 +39,38 @@ function onCreateWebpackConfig({ actions, loaders }) {
       ],
     },
   })
+
+  if (stage === `develop`) {
+    let ts
+    try {
+      ts = require.resolve(`typescript`)
+    } catch (e) {
+      reporter.warn(
+        `"typescript" is not installed. Builtin ESLint won't be working on typescript files.`
+      )
+    }
+
+    if (ts) {
+      const builtInEslintRule = getConfig().module.rules.find(rule => {
```

Comment:
```suggestion
      const hasBuiltInEslintRule = getConfig().module.rules.find(rule => {
```

Reply:
Since `builtInEslintRule` is used to create `typescriptEslintRule` not only for the conditional check. I prefer to stick with `builtInEslintRule`.

Reply:
Oh you're right :) builtInEslintRule makes more sense, I missed that part  👍